examples/http-request-errors.yaml

```diff
@@ -1,32 +1,28 @@
 alerts:
 - expr: |
     (
-      sum(status_class:promhttp_metric_handler_requests_total:rate5m{status_class="5xx"})
+      sum(rate_5m_by_code:promhttp_metric_handler_requests_total{status_class="5xx"})
     /
-      sum(status_class:promhttp_metric_handler_requests_total:rate5m)
+      sum(rate_5m_by_code:promhttp_metric_handler_requests_total)
     ) > 0.050000000000000003
   for: 5m
   labels:
-    job: fooapp
-    namespace: default
     severity: warning
 - expr: |
     (
-      sum(status_class:promhttp_metric_handler_requests_total:rate5m{status_class="5xx"})
+      sum(rate_5m_by_code:promhttp_metric_handler_requests_total{status_class="5xx"})
```

Comment:
hm I am not sure about this one. Does the recording rule add this `status_class` label?